app/controllers/users/sessions_controller.rb

```diff
@@ -74,10 +74,21 @@ def increment_session_bad_password_count
     def process_locked_out_session
       warden.logout(:user)
       warden.lock!
-      flash[:error] = t('errors.sign_in.bad_password_limit')
+
+      flash[:error] = t(
+        'errors.sign_in.bad_password_limit',
+        time_left: locked_out_time_remaining,
+      )
       redirect_to root_url
     end
 
+    def locked_out_time_remaining
+      locked_at = session[:max_bad_passwords_at]
+      window = IdentityConfig.store.max_bad_passwords_window_in_seconds.seconds
+      new_time = Time.zone.at(locked_at) + window
```

Comment:
What does "new" refer to in `new_time` ? Should we call this something referring to the expiration time?